frontend/src/components/organizations/OrganizationInfo.vue

```diff
@@ -216,55 +221,95 @@ export default {
         return {}
       },
     },
-    loading: {
+    uploadedDocuments: {
+      type: Array,
+      default: () => {
+        return []
+      },
+    },
+    loadingAll: {
+      type: Boolean,
+      default: false,
+    },
+    loadingInclusionPolicy: {
```

Comment:
If we need to keep this flag, I don't think it needs to be a prop. You can just manage it within the save method directly.

Reply:
Fixed. re: initalizeData() with the new changes it clearly revealed we need to reset documents. Not sure why was working before. Regardless fixed.